utils/formatTable.ts

```diff
@@ -40,18 +41,17 @@ export default (data: DataType[]) => {
     headers,
     datasets: []
   }
-  data.forEach(d => {
-    const TableRow: TableDataType = {
-      公表日: dayjs(d['リリース日']).format('MM/DD') ?? '不明',
-      居住地: d['居住地'] ?? '調査中',
-      年代: d['年代'] ?? '不明',
-      性別: d['性別'] ?? '不明',
-      退院: d['退院']
+  const datasets = data.map(
+    (d): TableDataType => {
+      return {
+        公表日: dayjs(d['リリース日']).format('MM/DD') ?? '不明',
+        居住地: d['居住地'] ?? '調査中',
+        年代: d['年代'] ?? '不明',
+        性別: d['性別'] ?? '不明',
+        退院: d['退院']
+      }
     }
-    tableDate.datasets.push(TableRow)
-  })
-  tableDate.datasets.sort((a, b) =>
-    a.公表日 === b.公表日 ? 0 : a.公表日 < b.公表日 ? 1 : -1
   )
+  tableDate.datasets = sortBy(datasets, [o => -dayjs(o.公表日)])
```

Comment:
JavaScript の `Array#sort` メソッドは並び順を保証しないので、実行エンジンごとに並び順が変わる可能性があります。

>  For a long time, the JavaScript spec didn’t require sort stability for Array#sort, and instead left it up to the implementation. And because this behavior was unspecified
ref: https://v8.dev/features/stable-sort

今回は SSR の Node.js v10  と CSR 時の Chrome で並び順が変わったことがエラーの原因だと思います。
対策として、lodash は並び順を保証した `sort` メソッドを持っているので、そちらでソートを実装しました。

> This method performs a stable sort, that is, it preserves the original sort order of equal elements
ref: https://lodash.com/docs#sortBy

TC37 には並び順を保証した `sort` が仕様にはいっていて、Chrome 70 や Node.js 12+ で実装されているようですが、それ以外のクライアントのブラウザ環境を考えると lodash の sort で実装を揃えておくのが安全かなと思います。